app/src/main/java/org/wikipedia/readinglist/ReadingListBehaviorsUtil.kt

```diff
@@ -149,7 +149,7 @@ object ReadingListBehaviorsUtil {
         FeedbackUtil
                 .makeSnackbar(activity,
                         activity.getString(
-                                if (lists.size == 1) R.string.reading_list_item_deleted else R.string.reading_lists_item_deleted, page.title()),
+                                if (lists.size == 1) R.string.reading_list_item_deleted_from_list else R.string.reading_lists_item_deleted, page.title()),
```

Comment:
The first one will cause a crash since it has two parameters.

Reply:
Thank you for catching this @cooltey. Done.